src/project-management/release-train-model.md

```diff
@@ -0,0 +1,55 @@
+# Release Train Model
+
+## What is the release train model?
+
+Before introducing the concept of the release train model, let us take a review of the delivery mode of TiDB in the past.
+
+In releases earlier than v5.0, the release frequency of TiDB major versions was a year or half a year, which is quite a long development cycle. The long development cycle has both benefits and drawbacks as follows:
+
+- Benefits: the longer a development cycle is, the more features one release can deliver.
+- Drawbacks: the longer a development cycle is, the more difficulties we have to coordinate regression and acceptance tests, and the more possibly a delay happens. Also, if new feature requests are received during the long development cycle, these new features are added to the development backlog after the start of the development cycle. In this case, development tasks are hardly converged before the release date.
+
+Starting from v5.0, TiDB adopts the release train model, which is a product development model for requirements gathering, analysis, decision making, release, and issue feedback.
+
+Just like a train delivering goods, decisions need to be made about the priorities of the goods, destination, arrival time, which train to load on, which carriage, etc., before the train departs.
+
+The benefits of moving to the release train model are as follows:
+
+1. A shorter feedback cycle: users can benefit from features shipped faster.
+2. Easier predictability for contributors and users:
+    1. Developers and reviewers can decide in advance the target release to deliver specific features.
+    2. If a feature misses a release train we have a good idea of when it will show up.
+    3. Users know when to expect their features.
+3. Transparency. There will be a published cut-off date (AKA code freeze) for the release and people will know about it in advance. Hopefully this will remove the contention around which features make it.
```

Comment:
```suggestion
3. Transparency. There will be a published cut-off date (AKA code freeze) for the release and people will know about the date in advance. Hopefully this will remove the contention around which features will be included in the release.
```

what about this @qiancai ?

Reply:
LGTM